app/lib/screens/history/PdfPreview.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_full_pdf_viewer/flutter_full_pdf_viewer.dart';
+
+class PdfPreview extends StatelessWidget {
```

Comment:
- [ ] The filename should be lowercase `pdfpreview.dart` or `pdf_preview.dart`

Reply:
@holtzmak I changed the file name. Thank you.

Reply:
I do not see a change in the file name?

Reply:
I did change the file name, I don't know why it is not showing. 
![image](https://user-images.githubusercontent.com/44125953/104678437-123b2c00-56b1-11eb-8a01-8799ccbf32ab.png)